src/config/chains/telos-zkevm-testnet/index.ts

```diff
@@ -301,6 +302,10 @@ export default class TelosZkEVM extends EVMChainSettings {
         return config.headerIndicators;
     }
 
+    getIndexerSecondsBehindThreshold(): number {
+        return config.secondsBehindThreshold || 3;
```

Comment:
Let's remove the default and just throw an error if it isn't properly configured, what do you think?